atc/atccmd/command.go

```diff
@@ -1172,6 +1172,8 @@ func (cmd *RunCommand) backendComponents(
 func (cmd *RunCommand) compression() compression.Compression {
 	if cmd.StreamingArtifactsCompression == "zstd" {
 		return compression.NewZstdCompression()
+	} else if cmd.StreamingArtifactsCompression == "nozip" {
```

Comment:
`nozip` is confusing IMHO, how about `uncompressed`? Or `raw`?

Reply:
I prefer `raw` here. WDYT @evanchaoli ?

Reply:
I am ok.